ui/timer.qml

```diff
@@ -183,9 +191,11 @@ Mycroft.Delegate {
                     id: timeRemaining
                     anchors.horizontalCenter: timerBackground.horizontalCenter
                     anchors.verticalCenter: parent.verticalCenter
+                    horizontalAlignment: Text.AlignHCenter
+                    width: timerBackground.width - Kirigami.Units.largeSpacing
                     color: "white"
                     font.family: "Noto Sans"
-                    font.pixelSize: parent.height * 0.2
+                    font.pixelSize: contentWidth > timerBackground.width ? parent.height * 0.10 : parent.height * 0.18
```

Comment:
This looked a little small, especially compared to the timer name, can we bump them up a little to 0.15 and 0.25?